### DISCUSSION TITLE
Humenize the click - random delay between clicks

### DISCUSSION THREAD

**Discussion by orrans** [felladrin/linkedin-autoconnect-chrome-extension#12] [Ideas]
Can you make the clicks more humanized?
add some random delays between the connect invites clicks

**Comment by felladrin** [comment]
Hi, @orrans. This has been implemented, and now it has a random delay between 1.5 and 3 seconds before clicking the Connect Button again.
Please check the [latest release](https://github.com/felladrin/linkedin-autoconnect-chrome-extension/releases/latest) and let me know what you think.
